cg/services/analysis_starter/configurator/file_creators/nextflow/params_file/abstract.py

```diff
@@ -1,13 +1,15 @@
 from abc import ABC, abstractmethod
 from pathlib import Path
 
+from cg.constants import FileExtensions
+
 
 class ParamsFileCreator(ABC):
 
     @staticmethod
-    @abstractmethod
     def get_file_path(case_id: str, case_path: Path) -> Path:
-        pass
+        """Return the path to the params file."""
+        return Path(case_path, f"{case_id}_params_file").with_suffix(FileExtensions.YAML)
```

Comment:
Hmm, I want to skip `case_id_` and `file`here. Not in this PR but maybe in the future. Any opinions?

Reply:
Future crusade